test/plugins/commands.test.js

```diff
@@ -154,46 +141,46 @@ describe('commands plugin test', () => {
             options.url = '/commands/foo/bar/1.2.3';
             options.credentials.scope = ['user'];
 
-            return server.inject(options).then((reply) => {
-                assert.equal(reply.statusCode, 403);
+            return server.inject(options).then((response) => {
+                assert.equal(response.statusCode, 403);
             });
         });
 
         it('returns 5xx if cache is bad', () => {
             options.url = `/commands/${mockCommandNamespace}/`
                 + `${mockCommandName}/${mockCommandVersion}`;
             // @note this pushes the payload size over the 512 byte limit
-            options.payload += 'WEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE';
-            options.payload += 'WEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE';
+            options.payload += 'REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE';
```

Comment:
What is "REE"? Is "REE" > "WEE"

Reply:
Oh, strange, dunno how that got there :^)

According to JavaScript,

```js
'REE' > 'WEE' === false
```

But it's arbitrary, so we might as well leave it 😄